docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst

```diff
@@ -27,7 +27,7 @@ due to cervical cancer increase over age, and have the highest value 29 per
      - ICD10
      - IDC9
```

Comment:
Since we didn't use ICD9 codes in calculation, is it ok to remove the ICD9 column?

Reply:
yes, please remove

Reply:
Will do!